README.md

```diff
@@ -12,23 +12,25 @@ The Modules
+| <a href="https://github.com/tlaplus/CommunityModules/blob/master/modules/Relation.tla">Relation.tla</a> | Basic operations on relations, represented as binary Boolean functions over some set S.| | [@muenchnerkindl](https://github.com/muenchnerkindl) |
+| <a href="https://github.com/tlaplus/CommunityModules/blob/master/modules/FiniteSetsExt.tla">FiniteSetsExt.tla</a> | An Operator to do folds without having to use RECURSIVE. | &#10004;| [@hwayne](https://github.com/hwayne),[@lemmy](https://github.com/lemmy) |
+| <a href="https://github.com/tlaplus/CommunityModules/blob/master/modules/Bitwise.tla">Bitwise.tla</a> | Bitwise And and shift-right. | [&#10004;](https://github.com/tlaplus/CommunityModules/blob/master/modules/tlc2/overrides/Bitwise.java) | [@lemmy](https://github.com/lemmy),[@pfeodrippe](https://github.com/pfeodrippe) |
+| <a href="https://github.com/tlaplus/CommunityModules/blob/master/modules/DifferentialEquations.tla">DifferentialEquations.tla</a> | see page 178ff of [Specifying Systems](https://lamport.azurewebsites.net/tla/book-02-08-08.pdf)| | Leslie Lamport |
+| <a href="https://github.com/tlaplus/CommunityModules/blob/master/modules/Json.tla">Json.tla</a> | Print TLA+ values as JSON values. | [&#10004;](https://github.com/tlaplus/CommunityModules/blob/master/modules/tlc2/overrides/Json.java)| [@kuujo](https://github.com/kuujo) |
+| <a href="https://github.com/tlaplus/CommunityModules/blob/master/modules/SVG.tla">SVG.tla</a> | see https://github.com/will62794/tlaplus_animation | [&#10004;](https://github.com/tlaplus/CommunityModules/blob/master/modules/tlc2/overrides/SVG.java)| [@will62794](https://github.com/will62794), [@lemmy](https://github.com/lemmy) |
+| <a href="https://github.com/tlaplus/CommunityModules/blob/master/modules/ShiViz.tla">ShiViz.tla</a> | Visualize error-traces of multi-process PlusCal algorithms with an [Interactive Communication Graphs](https://bestchai.bitbucket.io/shiviz/). |  | [@lemmy](https://github.com/lemmy) |
+| <a href="https://github.com/tlaplus/CommunityModules/blob/master/modules/TLCExt.tla">TLCExt.tla</a> | Assertion operators and experimental TLC features. | [&#10004;](https://github.com/tlaplus/CommunityModules/blob/master/modules/tlc2/overrides/TLCExt.java)| [@lemmy](https://github.com/lemmy), [@will62794](https://github.com/will62794) |
+| <a href="https://github.com/tlaplus/CommunityModules/blob/master/modules/IOUtils.tla">IOUtils.tla</a> | Input/Output of TLA+ values & Spawn system commands from a spec. | [&#10004;](https://github.com/tlaplus/CommunityModules/blob/master/modules/tlc2/overrides/IOUtils.java) | [@lemmy](https://github.com/lemmy), [@lvanengelen](https://github.com/lvanengelen) |
 
 
 How to use it
 -------------
 
+You must be running Java 9 or higher.
```

Comment:
Why Java 9?  Gson seems to work fine with Java 1.8 here.

Reply:
The problem is at `IOUtils.java` which uses `readAllBytes`, it was introduced only at Java 9, but maybe I'm configuring the JVM in a invalid way? Let me know.

Running
```shell
$ export JAVA_HOME=/usr/lib/jvm/java-1.8.0-openjdk-amd64
$ ant test
```

returns

```java
...
compile:
    [javac] Compiling 9 source files to /home/rafael/dev/CommunityModules/build/modules
    [javac] /home/rafael/dev/CommunityModules/modules/tlc2/overrides/IOUtils.java:89: error: cannot find symbol
    [javac] 		final StringValue stdout = new StringValue(new String(process.getInputStream().readAllBytes()));
    [javac] 		                                                                              ^
    [javac]   symbol:   method readAllBytes()
    [javac]   location: class InputStream
    [javac] /home/rafael/dev/CommunityModules/modules/tlc2/overrides/IOUtils.java:90: error: cannot find symbol
    [javac] 		final StringValue stderr = new StringValue(new String(process.getErrorStream().readAllBytes()));
    [javac] 		                                                                              ^
    [javac]   symbol:   method readAllBytes()
    [javac]   location: class InputStream
    [javac] /home/rafael/dev/CommunityModules/modules/tlc2/overrides/IOUtils.java:130: error: cannot find symbol
    [javac] 		final StringValue stdout = new StringValue(new String(process.getInputStream().readAllBytes()));
    [javac] 		                                                                              ^
    [javac]   symbol:   method readAllBytes()
    [javac]   location: class InputStream
    [javac] /home/rafael/dev/CommunityModules/modules/tlc2/overrides/IOUtils.java:131: error: cannot find symbol
    [javac] 		final StringValue stderr = new StringValue(new String(process.getErrorStream().readAllBytes()));
    [javac] 		                                                                              ^
    [javac]   symbol:   method readAllBytes()
    [javac]   location: class InputStream
    [javac] 4 errors
```

Reply:
Oh, good catch!  Can you please open a new issue for IOUtils and another one to configure the GitHub action to run with 1.8?

Reply:
Opened both issues, thanks!